js/bitmax.js

```diff
@@ -197,39 +254,41 @@ module.exports = class bitmax extends Exchange {
     async fetchMarkets (params = {}) {
         const response = await this.publicGetProducts (params);
         //
-        //     [
-        //         {
-        //             "symbol" : "BCH/USDT",
-        //             "domain" : "USDS",
-        //             "baseAsset" : "BCH",
-        //             "quoteAsset" : "USDT",
-        //             "priceScale" : 2,
-        //             "qtyScale" : 3,
-        //             "notionalScale" : 9,
-        //             "minQty" : "0.000000001",
-        //             "maxQty" : "1000000000",
-        //             "minNotional" : "5",
-        //             "maxNotional" : "200000",
-        //             "status" : "Normal",
-        //             "miningStatus" : "",
-        //             "marginTradable" : true,
-        //             "commissionType" : "Quote",
-        //             "commissionReserveRate" : 0.0010000000
-        //         },
-        //     ]
+        // {
+        //    "code": 0,
+        //    "data": [
+        //        {
+        //            "symbol": "DAD/USDT",
+        //            "baseAsset": "DAD",
+        //            "quoteAsset": "USDT",
+        //            "status": "Normal",
+        //            "minNotional": "5",
+        //            "maxNotional": "50000",
+        //            "marginTradable": False,
+        //            "commissionType": "Quote",
+        //            "commissionReserveRate": "0.001",
+        //            "tickSize": "0.00001",
+        //            "lotSize": "1"
```

Comment:
It would be a lot easier, if the exchange API returned the type of the market here (whether it's a contract or not), otherwise it's harder to distinguish `BTC-PERP` from the rest of the markets...

Reply:
Yes, it's not so convenient, but this api could help to decide
GET /api/pro/v1/futures/contracts

https://bitmax-exchange.github.io/bitmax-futures-api-doc/#futures-contracts